src/main/java/Deadline.java

```diff
@@ -0,0 +1,13 @@
+public class Deadline  extends Task{
+    private final String by;
```

Comment:
final variable should follow SCREAMING_SNAKE_CASE. I notice similar mistakes in other files as well (e.g. src/main/java/Event.java)